asyncio_mqtt/client.py

```diff
@@ -104,16 +105,17 @@ def __init__(
 )
 
 P = ParamSpec("P")
+S = TypeVar("S", bound="Client")
```

Comment:
Let's call it `ClientT` to make the use case clearer.

Reply:
Done.

Reply:
Hold on, this push failed, I'm on my way, back to push again